pkg/resources/tag_masking_policy_association_acceptance_test.go

```diff
@@ -22,86 +27,18 @@ func TestAcc_TagMaskingPolicyAssociationBasic(t *testing.T) {
 		CheckDestroy: nil,
 		Steps: []resource.TestStep{
 			{
-				Config: tagAttachmentConfig(accName, acc.TestDatabaseName, acc.TestSchemaName),
+				Config: tagAttachmentConfig(accName, acc.TestDatabaseName, acc.TestSchemaName, tag.ID()),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("snowflake_tag_masking_policy_association.test", "masking_policy_id", fmt.Sprintf("%s.%s.%s", acc.TestDatabaseName, acc.TestSchemaName, accName)),
-					resource.TestCheckResourceAttr("snowflake_tag_masking_policy_association.test", "tag_id", fmt.Sprintf("%s.%s.%s", acc.TestDatabaseName, acc.TestSchemaName, accName)),
+					resource.TestCheckResourceAttr("snowflake_tag_masking_policy_association.test", "tag_id", tag.ID().FullyQualifiedName()),
 				),
 			},
 		},
 	})
 }
 
-func TestAcc_TagMaskingPolicyAssociationsystem_functions_integration_testComplete(t *testing.T) {
```

Comment:
Was this test replaced with some other test? Why was it removed?

Reply:
It was removed because these tests would test the same thing now. Note that now we manage masking policies in `tag` resource and this one will be removed soon anyway.

Reply:
Yes, but it has not been removed yet, right? It is only deprecated, so we should not remove the test for it just yet.

Reply:
Reverted.